MAUI/Calendar/getting-started.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: Getting started with MAUI Calendar control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Calendar(SfCalendar) control and its basic features.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Getting started with .NET MAUI Calendar (SfCalendar)
+This section explains the steps required to add the Calendar control. This section covers only basic features needed to get started with Syncfusion Calendar.
```

Comment:
This section explains how to add the Calendar control. This section covers only the basic features needed to get started with Syncfusion Calendar.

Reply:
Content modified